dll/win32/syssetup/wizard.c

```diff
@@ -390,89 +391,217 @@ AckPageDlgProc(HWND hwndDlg,
     return FALSE;
 }
 
+static const WCHAR s_szProductOptions[] = L"SYSTEM\\CurrentControlSet\\Control\\ProductOptions";
+static const WCHAR s_szRosVersion[] = L"SYSTEM\\CurrentControlSet\\Control\\ReactOS\\Settings\\Version";
+static const WCHAR s_szWindowsNT[] = L"SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion";
+static const WCHAR s_szControlWindows[] = L"SYSTEM\\CurrentControlSet\\Control\\Windows";
+static const WCHAR s_szCurrentBuildNumber[] = L"CurrentBuildNumber";
+static const WCHAR s_szCurrentVersion[] = L"CurrentVersion";
+
 static BOOL
-DoWriteProductOption(PRODUCT_OPTION nOption)
+WriteProductOptionServer(void)
```

Comment:
You could simplify the code and combine this function with the other one for Workstation, if you also had an array like:
```
{
    {L"ServerNT", L"Terminal Server", L"5.2", L"3790", 0x200, TRUE},
    {L"WinNT", L"", L"5.1", L"2000", 0x300, TRUE} 
}
```
of versions with data.

Reply:
OK.